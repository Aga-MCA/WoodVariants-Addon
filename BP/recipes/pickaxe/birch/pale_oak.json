{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "birch_pale_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:pale_oak_stick"
      },
      "P": {
        "item": "aga_wood_variants:birch_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:birch_pale_oak_pickaxe"
    }
  }
}