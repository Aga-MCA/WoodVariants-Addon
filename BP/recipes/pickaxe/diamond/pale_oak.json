{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "diamond_pale_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:pale_oak_stick"
      },
      "P": {
        "item": "minecraft:diamond"
      }
    },
    "result": {
      "item": "aga_wood_variants:diamond_pale_oak_pickaxe"
    }
  }
}