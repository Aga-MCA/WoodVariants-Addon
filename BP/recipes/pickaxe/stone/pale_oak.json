{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "stone_pale_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:pale_oak_stick"
      },
      "P": {
        "item": "minecraft:cobblestone"
      }
    },
    "result": {
      "item": "aga_wood_variants:stone_pale_oak_pickaxe"
    }
  }
}