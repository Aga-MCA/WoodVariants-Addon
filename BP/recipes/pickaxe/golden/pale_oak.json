{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_pale_oak_pickaxe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PPP",
      " S ",
      " S "
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:pale_oak_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_pale_oak_pickaxe"
    }
  }
}